### PR TITLE
Qm bridge implementation 1

### PR DESCRIPTION
**Pull request by gulyx**
Implementation of the bridge for the QM and 
prototype implementation of the STUB for the QM Component

[Review on Reviewable](https://reviewable.io/reviews/learnpad/learnpad/192)

### CODE REVIEW

File: lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/me/XwikiController.java [hunk 1]
```diff
@@ -180,6 +185,9 @@ public MVResults checkModelSetVerification(String verificationProcessId)
 					// TODO Auto-generated catch block
 					e.printStackTrace();
 				}
+				// Notify QM about a new model set imported
+				String fooModelContent = "-ask-to-jean-how-to-retraive-the-modelset-";
+				this.qm.importModelSet(toReturn.getModelSetId(), toReturn.getType(), fooModelContent.getBytes());
```
Comment (woshilapin) [inline lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/me/XwikiController.java]: You should look into other components that do exactly the same operation. I guess that the following piece of code should help you.
https://github.com/LearnPAd/learnpad/blob/master/lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/cw/XwikiController.java#L131

File: lp-platform/lp-platform-distributions/lp-platform-jetty-hsqldb/pom.xml [hunk 2]
```diff
@@ -60,6 +60,18 @@
 			<type>jar</type>
 		</dependency>
 		<dependency>
+    		<groupId>org.xwiki.platform</groupId>
```
Comment (woshilapin) [inline lp-platform/lp-platform-distributions/lp-platform-jetty-hsqldb/pom.xml]: Why do you need this 2 dependencies?  At least, I don't see how they're related to this PR.

**Comment by woshilapin** [comment]
What about this Pull Request?

**Comment by gulyx** [comment]
well this is almost done ... i am testing it a bit more.
wait a bit @woshilapin

**Pull request closed without merging**